src/ApiSchema.ts

```diff
@@ -95,23 +103,34 @@ export class APISchema {
 
             const schema = await response.json();
 
-            const baseUrlRegex = this.constructBaseUrlRegex(schema);
-            const invalidBaseUrlRegex = this.constructInvalidBaseUrlRegex(schema);
-
+            this.platforms = schema.servers[0].variables.platform.enum;
             this.paths = [];
 
             for (const pathName in schema.paths) {
                 const pathSchema = schema.paths[pathName];
                 const methodSchema = pathSchema.get ? pathSchema.get : pathSchema.post;
 
                 if (!methodSchema) continue; // Only handle GET/POST
-                if (methodSchema.operationId.startsWith("tournament")) continue;
+                if (methodSchema.operationId.startsWith("tournament-v3")) continue;
 
                 const path = new Path();
+                path.name = pathName;
                 path.methodType = pathSchema.get ? "GET" : "POST";
-                path.method = methodSchema.operationId;
-                this.constructRegex(path, baseUrlRegex, invalidBaseUrlRegex, pathName, methodSchema);
-                console.assert(path.regex, "Path Regex should be set");
+                path.regex = this.constructRegex(pathName, methodSchema);
+
+                path.queryParameters = new Map();
+                path.pathParameters = new Map();
+                if (methodSchema.parameters) {
+                    for (const parameter of methodSchema.parameters) {
+                        if (parameter.in === "path") {
+                            this.addParameter(path.pathParameters, parameter);
```

Comment:
imo Path.addParameter(..) makes more sense than this.addParameter(..), especially from an oop standpoint